### PR TITLE
add template

### LINKED ISSUE #94: add label template
Reinstate the label template functionality in the custom worldmap panel.
This was added and exists in the editor html template.. but the functionality has gone.

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #94 
quick reinstatement of the tooltip label template change

### CODE REVIEW

File: custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts [hunk 1]
```diff
@@ -215,7 +215,13 @@ export default class WorldMap {
...
   createPopup(circle, locationName, value) {
     const unit = value && value === 1 ? this.ctrl.panel.unitSingular : this.ctrl.panel.unitPlural;
-    const label = (locationName + ': ' + value + ' ' + (unit || '')).trim();
+    const useTemplate = !!this.ctrl.panel.labelTemplate;
+    const label = useTemplate
+      ? this.ctrl.panel.labelTemplate
+          .replace('${name}', locationName)
+          .replace('${value}', value)
+          .replace('${unit}', unit)
+      : (locationName + ': ' + value + ' ' + (unit || '')).trim();
```
Comment (CarlosNZ) [inline custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts:224]: Just wondering, why not just:
```
const label = !!this.ctrl.panel.labelTemplate;
      ? this.ctrl.panel.labelTemplate
          .replace('${name}', locationName)
          .replace('${value}', value)
          .replace('${unit}', unit)
      : (locationName + ': ' + value + ' ' + (unit || '')).trim();
```
Just a stylistic/readability preference, or is there a difference?

**Comment by CarlosNZ** [comment]
Thanks for this. I'll add the same thing to #87 before merging too. (I already left a place-holder for it, just didn't enable it because I noticed it wasn't working.)

**Comment by mark-prins** [comment]
> `yarn build` -- update `dist` folder.

it was already done - the module.js and map are both in this PR already

File: custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts (hunk 1 above)
Comment (mark-prins) [reply custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts:224]: > Just a stylistic/readability preference, or is there a difference?

no difference - I've picked up the habit (from Will I think) of adding in variables in cases like this - it makes the intention clearer. I was also trying, not very successfully, to make the code block smaller

**Comment by CarlosNZ** [comment]
> > `yarn build` -- update `dist` folder.
> 
> it was already done - the module.js and map are both in this PR already

Oops. 👍

File: custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts (hunk 1 above)
Comment (CarlosNZ) [reply custom-plugins/grafana-worldmap-panel-msupply/src/worldmap.ts:224]: Cool -- I did it the other way on the regional map panel. ;)

**Pull request merged**